src/test_l1_dsDisplay.c

```diff
@@ -184,8 +184,14 @@ void test_l1_dsDisplay_negative_dsDisplayInit(void) {
 
     // Step 02: Attempt to initialize the display module again
     result = dsDisplayInit();
-    UT_LOG("\n In %s Return value: [%d]\n", __FUNCTION__, result);
-    DS_ASSERT_AUTO_TERM_NUMERICAL(result, dsERR_ALREADY_INITIALIZED);
+#ifdef ENABLE_ENHANCED_ERROR_CODE
```

Comment:
https://github.com/rdkcentral/ut-core/discussions/55 -> please update to support the KVP module , so we can commit this code as a valid upgrade, that can change configuration at runtime.